week1-IntroAndSetupGuide.md

```diff
@@ -0,0 +1,38 @@
+# Part 1
+This chapter will get you setup on your repository, and kick start your front end code.
+
+## Setting up
+
+As a preparation for this session you should have received a repository name (if you don't, let an instructor know), have git and some git client installed ([GitHub Desktop](https://desktop.github.com/) recommended). Also, please make sure you have read and completed all the [Requirements](https://github.com/codessintheclassroom/classroom-material/blob/master/REQUIREMENTS.md) for this workshop.
+
+With your repo name in hands (it will be your working repo for the duration of the project), let's get started! 
+
+ 1. Create a local folder to keep your code. Ex: C:\CodessInTheClassroom.
+ 2. Find your repo at [CodessInTheClassrom GitHub project](https://github.com/codessintheclassroom).
+ 3. `git clone` your repository.  
+   3.1. If you're using GitHub desktop, see [this page](https://help.github.com/en/desktop/contributing-to-projects/cloning-a-repository-from-github-desktop) for further instructions on cloning.
```

Comment:
add a brief description of how to clone a git repo for those that don't use the GitHub Desktop app - because it's not available on Linux, and some people will use Linux.
OR
we should probably have a brief git readme in which we collect all the useful commands and link them from every other readme

Reply:
I like the idea of a Git "cheatsheet" that we can reference in other docs, will go with that one.
Thanks!